LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentResult.kt

```diff
@@ -18,5 +18,5 @@ sealed class LocalPaymentResult {
     /**
      * The user canceled the local payment flow.
      */
-    object Cancel : LocalPaymentResult()
+    data object Cancel : LocalPaymentResult()
```

Comment:
Is this required to be a data object? What does it change?

Reply:
Studio suggested converting the sealed sub-object to a data object, but this change neither improved functionality nor caused any regression issues. Since data object is not a valid Kotlin construct, I will revert the sub-object back to an object.